cypress/support/pages/editorPage.ts

```diff
@@ -0,0 +1,34 @@
+class EditorPage {
+  public open(): void {
+    cy.visit("/editor");
+  }
+
+  private get titleField() {
```

Comment:
@karenn32 Why private? What in case if you need to use the getter in the step definitions?